src/docs/codelabs/index.md

```diff
@@ -124,6 +124,11 @@ Learn how to use Flutter with other technologies.
   and a rewarded ad to an app called Awesome Drawing Quiz,
   a game that lets players guess the name of the drawing.
 
+* [Adding AdMob banner and native inline ads to a Flutter app][]<br>
```

Comment:
LGTM, although I'm a little confused about why Ads is capitalized in the previous item, and what the difference is between this codelab and the one above it.